src/hotspot/share/prims/jvm.cpp

```diff
@@ -151,29 +151,22 @@
   and thus can only support use of handles passed in.
 */
 
-static void trace_class_resolution_impl(Klass* to_class, TRAPS) {
+extern void trace_class_resolution(Klass* to_class) {
```

Comment:
why `extern` ?

Reply:
jni.cpp functions call this.

Reply:
I don't see any difference in the callers in relation to this PR and the function is not presently declared `extern`. ??

Reply:
There was an extern trace_class_resolution() function that called this _impl function that I removed, so renamed this impl function to trace_class_resolution().

Reply:
You're right, it has extern linkage but not declared with extern.